elide-datastore/elide-datastore-hibernate3/src/main/java/com/yahoo/elide/datastores/hibernate3/HibernateTransaction.java

```diff
@@ -238,21 +196,48 @@ public Object getRelation(
                 @SuppressWarnings("unchecked")
                 Class<?> relationClass = dictionary.getParameterizedType(entity, relationName);
 
+                AbstractHQLQueryBuilder.Relationship relationship = new AbstractHQLQueryBuilder.Relationship() {
+                    @Override
+                    public Class<?> getParentType() {
+                        return entity.getClass();
+                    }
+
+                    @Override
+                    public Class<?> getChildType() {
+                        return relationClass;
+                    }
+
+                    @Override
+                    public String getRelationshipName() {
+                        return relationName;
+                    }
+
+                    @Override
+                    public Object getParent() {
+                        return entity;
+                    }
+
+                    @Override
+                    public Collection getChildren() {
+                        return filteredVal;
+                    }
+                };
+
```

Comment:
Is there a reason to use an inline anonymous class here? It would probably be cleaner to have a this trivial class generated by Lombok (e.g. `@Data`)